consensus/core/src/metrics.rs

```diff
@@ -190,6 +191,13 @@ impl NodeMetrics {
                 exponential_buckets(1.0, 1.4, 20).unwrap(),
                 registry,
             ).unwrap(),
+            block_ancestors_depth: register_histogram_vec_with_registry!(
+                "block_ancestors_depth",
+                "The depth in rounds of ancestors included in newly proposed blocks",
+                &["authority"],
+                exponential_buckets(1.0, 2.0, 14).unwrap(),
```

Comment:
It will be one of the largest cardinality metrics but it is probably ok. I wonder if just using the last depth is good enough. How does the graph look like in private testnet?

Reply:
I believe we do have already histograms by authorities (ex `suspended_block_time` ) so worst case scenario it will be as bad as that one. Haven't tried that yet in PT, will do today and see the results.